src/css/controls/imports/display.less

```diff
@@ -103,6 +103,8 @@ display icons
     }
 
     .jw-display .jw-icon {
+        padding: 0 (@mobile-touch-target * 0.5);
```

Comment:
Can you put the padding on `jw-display-icon-container` instead of `jw-icon` so that applying a background and border-radius to `jw-display-icon-container` results in a circle or square with symmetrical width and height.